src/lib/components/compositions/input-floating/floating-label.stories.js

```diff
@@ -30,6 +30,19 @@ export const Base = {
   },
 };
 
+export const WithValue = {
```

Comment:
Adding separate "checked" and "with value" states to the components, so they get screenshot and checked at every PR build.